packaging/container-deployment/Containerfile.bpfd

```diff
@@ -12,6 +10,9 @@ RUN apt-get update && apt-get install -y\
  gcc-multilib\
  musl-tools
 
+WORKDIR /usr/src/bpfd
```

Comment:
I like to install first, then do local work. That way when code changes and you rebuild, install layer is run from cache.

Reply:
Sweet that actually sounds like a nice development time saver :) 

Can you update the images for bpfctl and bpfd-agent in https://github.com/redhat-et/bpfd/tree/main/packaging/container-deployment as well just to keep everything consistent?

Reply:
Fixed in the next PR.

I was in Containerfile.bpfd because I was having trouble deploying and I moved it from scratch to an alpine base image so I could break into and poke around. I made the above change as well. Got it working and backed out the other changes. I think this went in by accident, but I do think it is a time saver.